proposals/records.md

```diff
@@ -79,18 +65,60 @@ override Equals(object o) => Equals(o as T);
 
 ### Copy and Clone members
 
-A record type contains two synthesized copying members if methods with the same
-signature are not already declared within the record type:
+A record type contains two synthesized copying members:
 
 * A protected constructor taking a single argument of the record type.
-* A public parameterless instance method called `Clone` which returns the record type.
+* A public parameterless virtual instance "clone" method with a compiler-reserved name
 
 The protected constructor is referred to as the "copy constructor" and the synthesized
-body copies the values of all instance fields declared in the input type to the corresponding 
+body copies the values of all accessible instance fields in the input type to the corresponding
 fields of `this`.
 
-The `Clone` method returns the result of a call to a constructor with the same signature as the
-copy constructor.
+The "clone" method returns the result of a call to a constructor with the same signature as the
+copy constructor. The return type of the clone method is the containing type, unless a virtual
+clone method is present in the base class. In that case, the return type is the current containing
+type if the "covariant returns" feature is supported and the override return type otherwise. The
+synthesized clone method is an override of the base type clone method if one exists. An error is
+produced if the base type clone method is sealed.
+
+## Positional record members
+
+In addition to the above members, records with a parameter list ("positional records") synthesize
+additional members with the same conditions as the members above.
+
+### Primary Constructor
+
+A record type has a public constructor whose signature corresponds to the value parameters of the
+type declaration. This is called the primary constructor for the type, and causes the implicitly
+declared default class constructor, if present, to be suppressed. It is an error to have a primary
+constructor and a constructor with the same signature already present in the class.
+
+At runtime the primary constructor
+
+1. executes the instance initializers appearing in the class-body
+
+1. invokes the base class constructor with the arguments provided in the `record_base` clause, if present
+
+
+### Properties
+
+For each record parameter of a record type declaration there is a corresponding public property
+member whose name and type are taken from the value parameter declaration.
+
+For a record:
+
+* A public `get` and `init` auto-property is created (see separate `init` accessor specification).
+  Each "matching" inherited abstract accessor is overridden. The auto-property is initialized to
```

Comment:
Do we require the `abstract` property to match the expected signature? Could an accessor be missing, or non-public? Could the property be a type that is assignable from the record parameter type?